### PR TITLE
feat: add resource request/limit support for adminer

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: main.star [hunk 1]
```diff
@@ -27,10 +40,18 @@ def run(plan, default_db, default_driver, default_password,
                 "ADMINER_USERNAME": default_username,
                 "ADMINER_AUTOLOGIN": "1",
             },
-        )
+            min_cpu=ADMINER_MIN_CPU,
+            max_CPU=ADMINER_MAX_CPU,
```
Comment (bharath-123) [inline main.star]: is this casing correct?

**Review by bharath-123 — APPROVED** [review]
thanks for the linting too!

File: main.star (hunk 1 above)
Comment (barnabasbusa) [reply main.star]: You are right, I was doing it incorrectly. Pls check again.

**Review by bharath-123 — APPROVED** [review]

**Comment by bharath-123** [comment]
Approved! Happy to review the changes at the Kurtosis end too :)

**Comment by barnabasbusa** [comment]
Please feel free to merge this, as this is now a requirement for my other PR to be merged

**Comment by bharath-123** [comment]
merging. I thought you had merge access.

**Comment by barnabasbusa** [comment]
This is your repo :D

**Pull request merged**